addons/website_event/static/tests/tours/website_event.js

```diff
@@ -6,7 +6,8 @@ odoo.define("website_event.tour", function (require) {
 
     var _t = core._t;
 
-    tour.register("event", {
+    tour.register("website_event_tour", {
```

Comment:
I don't understand all those tour changes ... especially that we have several tasks linked to it. I think we should keep tours in core apps, not move them in sub apps.

Reply:
We have a complete tour for the "Event" app, which contains the scope for this tour (it contains almost the same steps + more), so the task was to remove it. Then, I was told to keep it for the sake of testing, so I thought it would be better to rename it to give a better idea of the scope of the tour.
event tour : https://github.com/odoo/odoo/blob/master/addons/event/static/src/js/tours/event_tour.js

Reply:
Ok. Was not clear. If at the end we have a clean tour for event only, improved for website event, that's what we want.